client/src/components/miniComponents/CourseCards.jsx

```diff
@@ -0,0 +1,28 @@
+import "../../assets/css/courseCard.css";
+import placeholder from "../../assets/img/CookingCourse.png";
+
+export default function CourseCard(props) {
```

Comment:
Please either change the file name to CourseCard or the function name to CourseCards.